ui/app/AppLayouts/Wallet/stores/TokensStore.qml

```diff
@@ -0,0 +1,157 @@
+import QtQuick 2.15
+
+import SortFilterProxyModel 0.2
+import StatusQ 0.1
+
+import utils 1.0
+
+QtObject {
+    id: root
+
+    /* PRIVATE: Modules used to get data from backend */
+    readonly property var _walletModule: walletSectionNewModule
+    readonly property var _networksModule: networksModule
+
+    /* This contains the different sources for the tokens list
+       ex. uniswap list, status tokens list */
+    readonly property var sourcesOfTokensModel: SortFilterProxyModel {
+        sourceModel: root.dummySourcesList //root._walletModule.sourcesOfTokensModel
```

Comment:
Some TODO here?

Reply:
This change is part of the last commit that will be REMOVED once the PR is approved, since it's adding some dummy data and enables the `manage token` section just for testing the PR.